### ISSUE TITLE
POD test fails

### ISSUE THREAD

**Issue by eserte** [ikegami/perl-DateTime-Format-RFC3339#4]
t/05_pod.t in v1.4.0 fails like this:
```
#   Failed test 'POD test for blib/lib/DateTime/Format/RFC3339.pm'
#   at /usr/perl5.20.1Dp/lib/site_perl/5.20.1/Test/Pod.pm line 186.
# blib/lib/DateTime/Format/RFC3339.pm (256): =back without =over
# Looks like you failed 1 test of 1.
t/05_pod.t .................. 
Dubious, test returned 1 (wstat 256, 0x100)
Failed 1/1 subtests 
```

**Comment by xavierba** [comment]
I was about to report the same issue :-)

**Comment by ikegami** [comment]
I wanted to wrap things up before my vacation ended last night, and I only noticed the POD checker wasn't installed after I had already pushed everything. And I was like "What's the odd that I introduced a POD mistake?" sigh.

Fixed in 1.6.0.
